regimes/it/invoice_validator.go

```diff
@@ -109,15 +110,14 @@ func isItalianParty(party *org.Party) bool {
 }
 
 func validateAddress(value interface{}) error {
-	v, ok := value.([]*org.Address)
+	v, ok := value.(*org.Address)
 	if v == nil || !ok {
 		return nil
 	}
 	// Post code and street in addition to the locality are required in Italian invoices.
-	address := v[0]
-	return validation.ValidateStruct(address,
-		validation.Field(&address.Street, validation.Required),
-		validation.Field(&address.Code, validation.Required),
+	return validation.ValidateStruct(v,
+		validation.Field(&v.Street, validation.Required),
+		validation.Field(&v.Code, validation.Required),
```

Comment:
Technically the Code is only required for Italian addresses, it'll be replaced with `00000` for non-Italian addresses. Not sure if that is worth considering here.

Reply:
At the moment, this validation [applies only to Italian customers](https://github.com/invopop/gobl/blob/a07bb8222f5ea1280d9bbca1b1d2c3485544b0d6/regimes/it/invoice_validator.go#L95-L100). That check is not done for suppliers though. I'm guessing an assumption was made here that suppliers will always have an Italian address. Sounds like a fair assumption to me, but we can always move the `isItalianParty` condition to this very method to apply to both.